index.js

```diff
@@ -1,18 +1,26 @@
 module.exports = () => {
+  function processSize (propPrefix, decl, list) {
+    let sizes = list.space(decl.value)
+    if (sizes.length === 1) sizes[1] = sizes[0]
+
+    decl.cloneBefore({ prop: propPrefix + 'width', value: sizes[0] })
+    decl.cloneBefore({ prop: propPrefix + 'height', value: sizes[1] })
+
+    decl.remove()
+  }
+
   return {
     postcssPlugin: 'postcss-size',
     Declaration: {
-      size: (decl, { list }) => {
+      'size': (decl, { list }) => {
         if (decl.parent.name !== 'page') {
-          let sizes = list.space(decl.value)
-          if (sizes.length === 1) sizes[1] = sizes[0]
+          processSize('', decl, list)
+        }
+      },
 
-          decl.cloneBefore({ prop: 'width', value: sizes[0] })
-          decl.cloneBefore({ prop: 'height', value: sizes[1] })
+      'min-size': (decl, { list }) => processSize('min-', decl, list),
```

Comment:
```suggestion
      'min-size': (decl, { list }) => {
        processSize('min-', decl, list)
      },
```

Since we are not using `processSize()` return value